src/compiler.adoc

```diff
@@ -779,9 +779,8 @@ will be familiar with almost all the scripts that comes with *mies* template.
 
 ==== Managing dependencies
 
-The real purpose of use leiningen for the ClojureScript compilation process is managing
-dependencies. Because it makes this task more obvious in comparison to doing it
-manually.
+The real purpose of using Leiningen for the ClojureScript compilation process is to automate
+the retrieval of dependencies.  This is dramatically simpler than retrieving them manually.
```

Comment:
- verb tense: of use -> of using
- elaborate: manage dependencies -> automate retrieval of dependencies